tslint.json

```diff
@@ -0,0 +1,9 @@
+{
+  "defaultSeverity": "error",
+  "extends": "tslint-config-airbnb",
+  "jsRules": {},
+  "rules": {
+    "import-name": false,
```

Comment:
please check

Reply:
import-name rule requires matching of variable and file name